real-main/app/handlers/appsync/handlers.py

```diff
@@ -1422,4 +1422,5 @@ def find_users(caller_user, arguments, **kwargs):
         raise ClientException('Cannot submit more than 100 combined emails and phoneNumbers')
 
     user_ids = user_manager.find_users(caller_user, emails=emails, phones=phones)
+    caller_user.update_last_found_users_at(pendulum.now('utc'))
```

Comment:
In my opinion, either this

```suggestion
    caller_user.update_last_found_users_at(now=pendulum.now('utc'))
```

or this

```suggestion
    caller_user.update_last_found_users_at()
```

would be better. The reason is that the parameter accepted by `update_last_found_users()` is a keyword argument.

Although python allows you to provide values for keyword arguments using their position, I think using that is generally bad... python 3 added a way to explicitly forbid this but it makes all your function definitions three characters longer, so I didn't use it in this project.

To me, this it's an error in python's design that you can pass values  to keyword arguments using their position.

The style guide here: https://python-docs.readthedocs.io/en/latest/writing/style.html talks a bid about this if you want to read more on it.